spec/dor/workflow/client/workflow_routes_spec.rb

```diff
@@ -169,8 +169,48 @@
   end
 
   describe '#create_workflow_by_name' do
-    it 'need to write these specs', skip: 'need to write specs' do
-      # something
+    let(:mock_requestor) { instance_double(Dor::Workflow::Client::Requestor, request: nil) }
```

Comment:
hey, i wrote some tests for the default and new scenarios

Reply:
TDD is not dead!